core/mo-services-impl/ccsds-com-impl/src/main/java/esa/mo/com/impl/provider/ArchiveProviderServiceImpl.java

```diff
@@ -62,8 +62,8 @@
 
     private final ArchiveManager manager = new ArchiveManager(null);
     private MALProvider archiveServiceProvider;
-    private boolean initialiased = false;
-    private boolean running = false;
+    private boolean initialiased;
+    private boolean running;
```

Comment:
Same here. I prefer to have explicit initializations as it doesn't leave room for ambiguity.